wire/wire.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package basictracer_go.wire;
+option go_package = "wire";
+
+message TracerState {
```

Comment:
I'm going to wait for opentracing/opentracing.github.io#73 to resolve first, since it seems that the github timer is going to start soon